contracts/utils/structs/EnumerableSet.sol

```diff
@@ -130,6 +130,13 @@ library EnumerableSet {
         return set._values[index];
     }
 
+    /**
+     * @dev Return the entier set in an array
```

Comment:
We should update the documentation to warn people to only use this for off-chain querying and be mindful that it can cause out of gas errors.

Also note the typo in "entier" which should be "entire".